### PR TITLE
Use Flutter Hooks

### LINKED ISSUE #167: Consider using Flutter Hooks
To simplify lots of flutter code, we should consider using [Flutter Hooks](https://pub.dev/packages/flutter_hooks)

### PR DESCRIPTION
**Pull request by Benjamin-Frost**
Closes #167

### CODE REVIEW

**Review by kolioOtSofia — APPROVED** [review]

File: app/lib/profile/pages/profile.dart [hunk 1]
```diff
@@ -1,130 +1,12 @@
...
-                        orElse: () => null,
-                      ),
-                    )
-                ],
-              ),
-            ),
-          );
-        },
-      ),
+    return Center(
```
Comment (dieknolle3333) [inline app/lib/profile/pages/profile.dart:8]: :fire:

**Comment by dieknolle3333** [comment]
before reviewing it I wanted to get an overview about what flutter hooks do, so from a really-experienced-app-developer-perspective, is this article useful? https://medium.com/flutter-community/flutter-hooks-7754df814995 or are there better resources?

**Review by dieknolle3333 — APPROVED** [review]
as I said due to the overflow on the buttom of the page in my emulator I cannot get out of the onboarding when starting the app (will be fixed in other issue), so I trust that others check out that everything still looks as expected :)

**Review by Toalaah — APPROVED** [review]

File: app/lib/profile/pages/profile.dart [hunk 2]
```diff
@@ -1,130 +1,12 @@
...
+      child: Text('Profil'),
```
Comment (SebastianWagner2) [inline app/lib/profile/pages/profile.dart:9]: I like these changes a lot in particular

Comment (SebastianWagner2) [reply app/lib/profile/pages/profile.dart:9]: Unironically though, getting rid of useless code is the best

**Review by SebastianWagner2 — APPROVED** [review]

**Review by LeonHermann322 — APPROVED** [review]

**Review by DevSchmidtchen — APPROVED** [review]

**Pull request merged**